apps/src/sites/code.org/pages/public/teacher-dashboard/index.js

```diff
@@ -259,6 +259,10 @@ function main() {
       }
     );
 
+    $scope.teacher_managed_section = function (section) {
+      return ['email', 'word', 'picture'].indexOf(section.login_type) > -1;
```

Comment:
Do you know if this gets polyfilled by our Babel plugin?  I was a little worried about breaking IE11 but I could be totally wrong...